common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -77,37 +84,48 @@
             }
         ],
         "@typescript-eslint/consistent-type-definitions": [
-            "error"
+            "error",
+            "interface"
+        ],
+        "@typescript-eslint/consistent-type-imports": [
+            "warn",
+            {
+                "prefer": "type-imports"
+            }
         ],
         "@typescript-eslint/dot-notation": [
             "error"
         ],
-        "@typescript-eslint/explicit-function-return-type": [
+        "@typescript-eslint/eol-last": [
             "off"
         ],
+        "@typescript-eslint/explicit-function-return-type": [
+            "warn",
+            {
+                "allowConciseArrowFunctionExpressionsStartingWithVoid": false,
+                "allowDirectConstAssertionInArrowFunctions": true,
+                "allowExpressions": true,
+                "allowHigherOrderFunctions": true,
+                "allowTypedFunctionExpressions": true
+            }
+        ],
         "@typescript-eslint/explicit-member-accessibility": [
-            "off"
+            "error",
+            {
+                "accessibility": "explicit"
+            }
         ],
         "@typescript-eslint/explicit-module-boundary-types": [
-            "off"
-        ],
-        "@typescript-eslint/func-call-spacing": [
-            "off"
+            "error"
         ],
         "@typescript-eslint/indent": [
             "off"
         ],
-        "@typescript-eslint/interface-name-prefix": [
```

Comment:
interface-name-prefix is probably a good rule to have turned on (I thought it had been on?)

Reply:
This has never been on AFAIK. We don't seem to follow a naming standard today for interfaces. Are you suggesting we adopt one?